rick.py

```diff
@@ -618,16 +618,16 @@ def stacked_chart(data_in, xlab, lab1, lab2, **kwargs):
             data[['values1', 'values2']] = data[['values1', 'values2']].astype(int)
         for i in data['values2']:
             if i < 0.1*upper:
-                ax.annotate(str(format(round(i,precision), ',')), xy=(i+0.015*upper, j-0.05), ha = 'left', color = 'k', fontname = font.normal, fontsize=10)
+                ax.annotate(str(format(round(i,precision), ',')), xy=(i-0.015*upper, j-0.05), ha = 'right', color = 'w', fontname = font.normal, fontsize=10)

```

Comment:
found in vfh_analysis/projects/2022_vfh_emissions/analysis/report_support/rick.py (branch: i17_1923_data)
added by scann0n

Reply:
Interestingly, @scann0n didn't appear to use the `stacked_chart` for this reporting, so this might have been introduced elsewhere and then the rick copied over.